packages/components/test/make-localized-element.test.js

```diff
@@ -11,55 +9,35 @@ describe('formatVars', () => {
   it('formats the l10nVars variables for spreading into the Localized', () => {
     const l10nVars = { name: 'Alice', number: 123 };
     const expectedOutput = { $name: 'Alice', $number: 123 };
-    expect(formatVars(l10nVars)).to.deep.equal(expectedOutput);
+    expect(formatVars(l10nVars)).toEqual(expectedOutput);
   });
 });
 
 describe('makeLocalizedElement', () => {
   it('Makes an element wrapped in a Localized component', () => {
     const Component = makeLocalizedElement('h1');
     const wrapper = shallow(<Component l10nId="test" />);
-    const expectedOutput = (
-      <Localized id="test">
-        <h1 />
-      </Localized>
-    );
-    assert(wrapper.contains(expectedOutput));
+    expect(wrapper).toMatchSnapshot();
```

Comment:
👍 
I do kind of like the simple `expectedOutput` as a visual of what this component looks like. The snapshots themselves are hard to read. I think it's a reasonable tradeoff, though, for the thoroughness and simplicity of the test code.